### ISSUE TITLE
NO REAL ISSUE: Move and contribute to react-native-exit-app?

### ISSUE THREAD

**Issue by wumke** [jundat95/react-native-close-app#3]
Hi, I'm the autor of the more widely used react-native-exit-app package.
I assume your package is doing the same stuff... so my question is:
Are you willing to contribute to my package and deprecate this one in favor of mine?

Often I'm to busy to handle it all myself, for example npm releases are not done that often because I focus on keeping the master branch up to date rather than releasing on npm (big sorry to all the users, but hey... I need to work for my family and people that pay me first... ).

I use many open source packages, and often I see a lot of similar projects... why not team up and improve existing packages instead of duplicating a lot of code???
Let me know...

**Comment by jundat95** [comment]
Oke, I will edit README and talk to every body install your react-native-exit-app

**Comment by juanchoperezj** [comment]
Good call! I'll try to find time to check the code and see how can I help to improve the library.
